v1/account/all.swagger.json

```diff
@@ -390,7 +390,7 @@
         },
         "parameters": [
           {
-            "name": "AccountID",
+            "name": "accountID",
```

Comment:
why is AccountID sometimes transfered to `accountID` and sometimes to `account_id`?

Reply:
oh, if it's a separate parameter, it's `accountID`, but if it's a field in the "JSON object", it's `account_id`?

Reply:
yes it is.